certificates/key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Which size is this private RSA key? 2048 bits? Keep in mind that according to predictions RSA keys of 2048 bits will be cracked after year 2048 (https://crypto.stackexchange.com/questions/1978/how-big-an-rsa-key-is-considered-secure-today). If it is 4096 bits then it is no problem...